commands/info.go

```diff
@@ -21,47 +21,101 @@ var (
 	}
 )
 
+// infoArguments represents the arguments we can make use of in this command
+type infoArguments struct {
+	token       string
+	verbose     bool
+	apiEndpoint string
+}
+
+// defaultInfoArguments returns an infoArguments object populated by the user's
+// command line arguments
+func defaultInfoArguments() infoArguments {
+	return infoArguments{
+		token:       cmdToken,
+		verbose:     cmdVerbose,
+		apiEndpoint: cmdAPIEndpoint,
+	}
+}
```

Comment:
What is the reason for this thing to exist? It still uses global variables. I don't see tests which inject different values. Is there any plan for this?